project/KenshooMonitoring.scala

```diff
@@ -59,6 +59,8 @@ object KenshooMonitoringBuild extends Build {
     )
     .settings(publishAllArtefacts : _*)
     .settings(makePublicallyAvailableOnBintray := true)
-    .settings(resolvers += Resolver.bintrayRepo("hmrc", "releases"))
+    .settings(resolvers ++= Seq(
```

Comment:
I think you should only update your resolvers in plugins.sbt, only the artifactory resolver should be in here if required. not sure what the jcenterRepo resolver is for either, so that might be good here too, but should try to have it if neccessary in plugins.sbt.
and you still have resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/" in plugins.sbt